src/test/kotlin/uk/gov/justice/digital/assessments/repositories/QuestionSchemaRepositoryTest.kt

```diff
@@ -15,7 +15,7 @@ import javax.transaction.Transactional
   Sql(scripts = ["classpath:referenceData/before-test.sql"], config = SqlConfig(transactionMode = SqlConfig.TransactionMode.ISOLATED)),
   Sql(scripts = ["classpath:referenceData/after-test.sql"], config = SqlConfig(transactionMode = SqlConfig.TransactionMode.ISOLATED), executionPhase = Sql.ExecutionPhase.AFTER_TEST_METHOD)
 )
-class QuestionSchemaRepositoryTest(@Autowired val questionSchemaRepository: QuestionSchemaRepository) : IntegrationTest() {
+open class QuestionSchemaRepositoryTest(@Autowired val questionSchemaRepository: QuestionSchemaRepository) : IntegrationTest() {
```

Comment:
I don't understand why this needs to be open.

Reply:
It must be the linter or IDE because I don't recall changing it!